bridge/core/dom/document.cc

```diff
@@ -52,7 +52,7 @@ Element* Document::createElement(const AtomicString& name, ExceptionState& excep
     return MakeGarbageCollected<WidgetElement>(name, this);
   }
 
-  return MakeGarbageCollected<HTMLUnknownElement>(name, *this);
```

Comment:
Why remove this?

Reply:
Sorry, I misunderstand with spec. I will add it back.